starter-code/pizza.js

```diff
@@ -1 +1,79 @@
-// Write your Pizza Builder JavaScript in this file.
+$(document).ready(function() {
+
+let defaultPrice = 13; 
+
+let togglePep = $(".btn-pepperonni").click(() => {
+  $(".pep").toggle()
+  $(".btn-pepperonni").toggleClass("active")
+  $("#pepPrice").toggle(200);
+  if ($(".btn-pepperonni").hasClass("active")) {
+    defaultPrice ++
+  } else defaultPrice --
+  $("#total").text(defaultPrice);
+});
+
+let toggleGreen = $(".btn-green-peppers").click(() => {
+  $(".green-pepper").toggle()
+  $(".btn-green-peppers").toggleClass("active")
+  $("#mushPrice").toggle(200);
+  if ($(".btn-green-peppers").hasClass("active")) {
+    defaultPrice ++
+  } else defaultPrice --
+  $("#total").text(defaultPrice);
+});
+
+let toggleMush = $(".btn-mushrooms").click(() => {
+  $(".mushroom").toggle()
+  $(".btn-mushrooms").toggleClass("active")
+  $("#greenPrice").toggle(200);
```

Comment:
Here, you are changing the green pepper price when you a clicking the btn-mushrooms button.